common/experiment_utils.py

```diff
@@ -59,14 +59,23 @@ def get_trial_instance_name(experiment: str, trial_id: int) -> str:
     return 'r-%s-%d' % (experiment, trial_id)
 
 
+def get_cycle_file_name(basename, cycle: int, suffix: str = None) -> str:
+    """Returns a filename for a file that is relevant to a particular snapshot
+    cycle."""
+    filename = basename + '-' + ('%04d' % cycle)
+    if suffix is None:
+        return filename
+    return filename + suffix
+
+
 def get_corpus_archive_name(cycle: int) -> str:
     """Returns a corpus archive name given a cycle."""
-    return 'corpus-archive-%04d.tar.gz' % cycle
+    return get_cycle_file_name('corpus-archive', cycle, '.tar.gz')
```

Comment:
Would these files ever be anything other than .tar.gz archives? I think it's probably fine to leave it as-is, but it might be simpler to always append the extension and drop the suffix argument if so.

Reply:
Done.